assets/dev/js/editor/regions/navigator/elements/hooks/ui/base/base.js

```diff
@@ -0,0 +1,7 @@
+import After from 'elementor-api/modules/hooks/ui/after';
+
+export default class Base extends After {
+	getConditions() {
+		return $e.components.get( 'navigator' ).isOpen;
```

Comment:
The UX ground rules are that each time the user drops an element somewhere, all the tree list down to this element is toggled so the navigator can display the element right away. Considering this condition, when the navigator is open, it should work, the tree list down to the newly dropped element is toggled. But when the navigator is closed, it won't happen. Why is this important? Because of the edge case in which the user opens the navigator after such a drag&drop action. Generally, each navigator change has to happen even though the navigator is closed, so it will be ready to be displayed when the user pop up the navigator again.

Reply:
On open, it will re-render itself.